resources/todomvc/architecture-examples/vue/vue.config.js

```diff
@@ -1,7 +1,11 @@
 const { defineConfig } = require("@vue/cli-service");
+const webpack = require("webpack");
 module.exports = defineConfig({
     transpileDependencies: true,
     publicPath: "",
+    configureWebpack: {
+        plugins: [new webpack.EvalSourceMapDevToolPlugin({})],
+    },
```

Comment:
Indeed the eval plugin isn't great for production, see https://webpack.js.org/configuration/devtool/

Not sure how to fix #385 but I wouldn't do it this way :/